### ISSUE TITLE
Do not jump to offscreen cursors

### ISSUE THREAD

**Issue by Pitometsu** [magnars/multiple-cursors.el#190]
Is it possible when edit some sentence by real cursor not jump visually to fake cursors on each move?

**Comment by magnars** [comment]
Mine doesn't jump to fake cursors. Could you elaborate?

**Comment by Pitometsu** [comment]
`mc/mark-all-like-this-dwim`, than when move cursor it jump to each offscreen fake cursor.

**Comment by magnars** [comment]
Not on my machine(tm). What happens if you disable your other plugins?

**Comment by Pitometsu** [comment]
You right, there's some conflict, let me check

**Comment by Pitometsu** [comment]
Seems like it conflict with `helm-mode`.

**Comment by Pitometsu** [comment]
It jumps over the buffer if I call `mc/mark-all-words-like-this` from `helm-M-x`.

**Comment by Pitometsu** [comment]
https://github.com/emacs-helm/helm/issues/960

**Comment by jtamagnan** [comment]
@Pitometsu is this still an issue for you?

**Comment by Pitometsu** [comment]
It not jumping now (but I tested this time in terminal only). But I should select `mc/mark-all-like-this-dwim` twice one by one, and only after that `helm` hide. `mc` only work fine.

**Comment by jtamagnan** [comment]
So yes, it does work? I'm sorry but I had a bit of trouble understanding what you meant. Thank you

**Comment by Pitometsu** [comment]
@jtamagnan work now, yes. Not jumping anymore.
But I got another issue:
boolean
When I press `M-x` #=> `helm-M-X` open
Then select `mc/mark-all-words-like-this` #=> all desired items in buffer selected
But `helm-M-x` still open. #=> this a bug
And when I select `mc/mark-all-words-like-this` from `helm-M-x` again, #=> `helm-M-x` at last close, and items in buffer stay selected: all right, desired behavior.

Problem in that I need select `mc/mark-all-words-like-this` from `helm-M-x` twice.
But I use `helm` in `popwin`, it may be cause of issue. But other commands work well.

**Comment by Pitometsu** [comment]
Maybe, I should create another ticket for this?

**Comment by magnars** [comment]
Multiple cursors does not work well with M-x, as listed under
https://github.com/magnars/multiple-cursors.el#known-limitations

**Comment by Pitometsu** [comment]
@magnars should I report issue to `helm-mode` author, or it more like `multiple-cursors` issue?

**Comment by magnars** [comment]
Neither. We already have issues for it. It's not something I've been able
to fix. So don't use M-x with mc, instead use keybindings.
ons. 25. mai 2016 kl. 11.16 skrev Netsu <notifications@github.com>:

**Comment by Pitometsu** [comment]
mc looks like collections of hacks, but awesome hacks, thank you!

**Comment by jtamagnan** [comment]
Since the original issue is no longer an issue I am closing this.
